views/index.handlebars

```diff
@@ -3,47 +3,27 @@
         <div class="post-preview">
             <a href="post.html">
                 <h2 class="post-title">
-                    Man must explore, and this is exploration at its greatest
+                    ManMannnn must explore, and this is exploration at its greatest
                 </h2>
                 <h3 class="post-subtitle">
                     Problems look mighty small from 150 miles up
                 </h3>
             </a>
             <p class="post-meta">Posted by <a href="#">Start Bootstrap</a> on September 24, 2014</p>
         </div>
-        <hr>
-        <div class="post-preview">
-            <a href="post.html">
-                <h2 class="post-title">
-                    I believe every human has a finite number of heartbeats. I don't intend to waste any of mine.
-                </h2>
-            </a>
-            <p class="post-meta">Posted by <a href="#">Start Bootstrap</a> on September 18, 2014</p>
-        </div>
-        <hr>
-        <div class="post-preview">
-            <a href="post.html">
-                <h2 class="post-title">
-                    Science has not yet mastered prophecy
-                </h2>
-                <h3 class="post-subtitle">
-                    We predict too much for the next year and yet far too little for the next ten.
-                </h3>
-            </a>
-            <p class="post-meta">Posted by <a href="#">Start Bootstrap</a> on August 24, 2014</p>
-        </div>
-        <hr>
+         {{#each posts}}
         <div class="post-preview">
             <a href="post.html">
                 <h2 class="post-title">
-                    Failure is not an option
+                    {{this.title}}
```

Comment:
Why this.title? Title should be enough. Does this work for you?

Reply:
I think it was already there in the default file. I am not sure. I will have a look at that.